### ISSUE TITLE
Text tasks have empty labels

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/Panoptes-Front-End#6870]
https://github.com/zooniverse/Panoptes-Front-End/blob/ebfa8b33ef6ed8faa3ecf621c1d6ea5e8d3449d1/app/classifier/tasks/text/index.jsx#L109-L119

`<label>` here is empty, so the `<textarea>` has no label. The task instruction is actually being shown in a `<div>`, outside the label.

<img width="600" alt="Unlabelled text task textbox in the Firefox accessibility tree." src="https://github.com/zooniverse/Panoptes-Front-End/assets/59547/76d3392c-1e74-438c-9a91-a4d655552808">

Relevant WCAG success criteria:
- [Labels or instructions (level A)](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html).
- [Info and relationships (level A)](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html).

## Expected behaviour
Each text input should have a label. Clicking the label will select the text input and focus it. The label will show up as the text area's accessible name in the document accessibility tree.
